src/components/dropdown/dropdown.js

```diff
@@ -382,6 +382,7 @@ const Dropdown = Input(InputIcon(InputLabel(InputValidation(class Dropdown exten
     let nextVal;
 
     switch (ev.which) {
+      case 9: // tab key
       case 13: // return
```

Comment:
generally think these are easier to read single lined rather than falling through
```suggestion
      case 9: case 13: // tab key or return key
```